index.md

```diff
@@ -120,11 +120,11 @@ title: .NET Documentation
                                                 </div>
                                             </div>
                                             <div class="cardText">
-                                                <h3><a href="https://msdn.microsoft.com/library/w0x726c2(v=vs.110).aspx">.NET Framework Guide</a></h3>
-                                                <p><a href="https://msdn.microsoft.com/library/hh425099(v=vs.110).aspx">Get Started</a></p>
-                                                <p><a href="https://msdn.microsoft.com/library/hh156542(v=vs.110).aspx">Tutorials</a></p>
-                                                <p><a href="https://msdn.microsoft.com/library/54xbah2z(v=vs.110).aspx">Client Development</a></p>
-                                                <p><a href="https://msdn.microsoft.com/library/bb400852(v=vs.110).aspx">ASP.NET</a></p>
+                                                <h3><a href="/dotnet/articles/framework/">.NET Framework Guide</a></h3>
+                                                <p><a href="/dotnet/articles/framework/get-started/">Get Started</a></p>
+                                                <p><a href="/dotnet/articles/framework/development-guide">Tutorials</a></p>
```

Comment:
One more ? ...

This one goes to ...

*.NET Framework Development Guide*

... not that that's wrong if that's where it should go, but it is a little bit :zap: shocking :zap: to land on something titled so differently and not obviously a list of tutorials.